barretenberg/cpp/src/barretenberg/honk/instance/prover_instance.cpp

```diff
@@ -199,13 +199,6 @@ std::shared_ptr<typename Flavor::ProvingKey> ProverInstance_<Flavor>::compute_pr
     proving_key = std::make_shared<ProvingKey>(dyadic_circuit_size, num_public_inputs);
     construct_selector_polynomials<Flavor>(circuit, proving_key.get());
 
-    if constexpr (StandardFlavor<Flavor>) {
-        // Compute sigma polynomials (we should update that late)
-        compute_standard_honk_sigma_permutations<Flavor>(circuit, proving_key.get());
-        compute_standard_honk_id_polynomials<Flavor>(proving_key.get());
-        compute_first_and_last_lagrange_polynomials<Flavor>(proving_key.get());
-    }
-
     if constexpr (IsUltraFlavor<Flavor>) {
```

Comment:
Remove the if constexpr for Ultra here and everywhere else, they were here to be able to include both Standard and Ultra functionality in a single class and not necessary anymore